e2e/test_operator_upgrades_with_chaos/operator_upgrades_with_chaos_test.go

```diff
@@ -93,9 +94,7 @@ var _ = Describe("Operator Upgrades with chaos-mesh", Label("e2e", "pr"), func()
 			clusterSetup(beforeVersion)
 			Expect(fdbCluster.SetAutoReplacements(false, 20*time.Minute)).ToNot(HaveOccurred())
 			// Ensure the operator is not skipping the process because it's missing for to long
-			Expect(
-				fdbCluster.SetIgnoreMissingProcessesSeconds(5 * time.Minute),
-			).NotTo(HaveOccurred())
+			fdbCluster.SetIgnoreMissingProcessesSeconds(5 * time.Minute)
```

Comment:
why does this error for unified but not split? or is it just not meaningful to check the error?

Reply:
I updated the method to not wait for reconciliation as everything that this method does is updating a setting in the `FoundationDBCluster` spec which doesn't have a direct affect on the running state (or will not cause any replacements). That's the reason the error return value was removed.

Reply:
ooh I see, thanks for the clarification!